### PR TITLE
feat(sumologicschemaprocessor)!: deprecate in favor of sumologicprocessor

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

**Review by kkujawa-sumo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ The rest of the components in the table are pure upstream OpenTelemetry componen
...
-|              [httpcheck][httpcheckreceiver]              |            [tail_sampling][tailsamplingprocessor]            |                                        |                                              |                                       |
-|                    [iis][iisreceiver]                    |               [transform][transformprocessor]                |                                        |                                              |                                       |
-|               [influxdb][influxdbreceiver]               |                                                              |                                        |                                              |                                       |
+|                [haproxy][haproxyreceiver]                |              [`sumologic`][sumologicprocessor]               |                                        |                                              |                                       |
```
Comment (astencel-sumo) [inline README.md:96]:
```suggestion
|                [haproxy][haproxyreceiver]                |              [sumologic][sumologicprocessor]               |                                        |                                              |                                       |
```

File: examples/sumologic-windows.yaml [hunk 2]
```diff
@@ -68,8 +68,8 @@ processors:
   ## Configuration for Sumo Logic Schema Processor
```
Comment (astencel-sumo) [inline examples/sumologic-windows.yaml:68]:
```suggestion
  ## Configuration for Sumo Logic Processor
```

File: examples/sumologic.yaml [hunk 3]
```diff
@@ -68,8 +68,8 @@ processors:
   ## Configuration for Sumo Logic Schema Processor
```
Comment (astencel-sumo) [inline examples/sumologic.yaml:68]:
```suggestion
  ## Configuration for Sumo Logic Processor
```

File: packaging/msi/SumoLogic.wixext/SumoLogicTests/TestData/with-extensions-block.yaml [hunk 4]
```diff
@@ -67,8 +67,8 @@ processors:
   ## Configuration for Sumo Logic Schema Processor
```
Comment (astencel-sumo) [inline packaging/msi/SumoLogic.wixext/SumoLogicTests/TestData/with-extensions-block.yaml:67]:
```suggestion
  ## Configuration for Sumo Logic Processor
```

File: packaging/msi/SumoLogic.wixext/SumoLogicTests/TestData/without-extensions-block.yaml [hunk 5]
```diff
@@ -48,8 +48,8 @@ processors:
   ## Configuration for Sumo Logic Schema Processor
```
Comment (astencel-sumo) [inline packaging/msi/SumoLogic.wixext/SumoLogicTests/TestData/without-extensions-block.yaml:48]:
```suggestion
  ## Configuration for Sumo Logic Processor
```

File: pkg/processor/sumologicschemaprocessor/README.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 # Sumo Logic Schema Processor
 
+**Sumologic Schema processor is depracated. Please use [Sumologic processor][sumologicprocessor] instead.**
```
Comment (astencel-sumo) [inline pkg/processor/sumologicschemaprocessor/README.md:3]:
```suggestion
**Sumologic Schema processor is depracated. Please use [Sumo Logic processor][sumologicprocessor] instead.**
```

File: pkg/processor/sumologicschemaprocessor/README.md [hunk 7]
```diff
@@ -1,5 +1,8 @@
...
+**Refer to [upgrading document](/docs/upgrading.md#upgrading-to-v0890-sumo-0).**
+
 **Stability level**: Beta
```
Comment (astencel-sumo) [inline pkg/processor/sumologicschemaprocessor/README.md:6]:
```suggestion
**Stability level**: Deprecated
```

**Review by astencel-sumo — APPROVED** [review]

File: .changelog/1316.breaking.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+feat(sumologicschemaprocessor)!:deprecate in favor of sumologicprocessor
```
Comment (sumo-drosiek) [inline .changelog/1316.breaking.txt:1]: ### 
```suggestion
feat(sumologicschemaprocessor)!: deprecate in favor of sumologicprocessor
```

**Pull request merged**